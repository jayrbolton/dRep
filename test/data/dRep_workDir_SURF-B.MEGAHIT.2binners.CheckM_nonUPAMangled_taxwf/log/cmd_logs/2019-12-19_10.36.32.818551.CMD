nucmer --mum -p /home/sumin/kbsdk-workspace/dRep/test/data/res.dRep.txwf.uniq/data/ANImf_files/Bin.012.u.fasta/Bin.012.u.fasta_vs_Bin.018.fasta -c 65 -g 90 /home/sumin/kbsdk-workspace/dRep/test/data/binned_contig_files_3bins/Bin.012.u.fasta /home/sumin/kbsdk-workspace/dRep/test/data/binned_contig_files_8bins/Bin.018.fasta; delta-filter -r -q /home/sumin/kbsdk-workspace/dRep/test/data/res.dRep.txwf.uniq/data/ANImf_files/Bin.012.u.fasta/Bin.012.u.fasta_vs_Bin.018.fasta.delta > /home/sumin/kbsdk-workspace/dRep/test/data/res.dRep.txwf.uniq/data/ANImf_files/Bin.012.u.fasta/Bin.012.u.fasta_vs_Bin.018.fasta.delta.filtered
