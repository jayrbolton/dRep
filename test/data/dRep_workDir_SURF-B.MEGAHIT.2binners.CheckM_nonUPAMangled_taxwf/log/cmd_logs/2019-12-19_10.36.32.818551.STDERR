1: PREPARING DATA
2,3: RUNNING mummer AND CREATING CLUSTERS
# reading input file "/home/sumin/kbsdk-workspace/dRep/test/data/res.dRep.txwf.uniq/data/ANImf_files/Bin.012.u.fasta/Bin.012.u.fasta_vs_Bin.018.fasta.ntref" of length 2619992
# construct suffix tree for sequence of length 2619992
# (maximum reference length is 536870908)
# (maximum query length is 4294967295)
# process 26199 characters per dot
#....................................................................................................
# CONSTRUCTIONTIME /usr/local/bin/MUMmer3.23/mummer /home/sumin/kbsdk-workspace/dRep/test/data/res.dRep.txwf.uniq/data/ANImf_files/Bin.012.u.fasta/Bin.012.u.fasta_vs_Bin.018.fasta.ntref 1.33
# reading input file "/home/sumin/kbsdk-workspace/dRep/test/data/binned_contig_files_8bins/Bin.018.fasta" of length 2655129
# matching query-file "/home/sumin/kbsdk-workspace/dRep/test/data/binned_contig_files_8bins/Bin.018.fasta"
# against subject-file "/home/sumin/kbsdk-workspace/dRep/test/data/res.dRep.txwf.uniq/data/ANImf_files/Bin.012.u.fasta/Bin.012.u.fasta_vs_Bin.018.fasta.ntref"
# COMPLETETIME /usr/local/bin/MUMmer3.23/mummer /home/sumin/kbsdk-workspace/dRep/test/data/res.dRep.txwf.uniq/data/ANImf_files/Bin.012.u.fasta/Bin.012.u.fasta_vs_Bin.018.fasta.ntref 3.98
# SPACE /usr/local/bin/MUMmer3.23/mummer /home/sumin/kbsdk-workspace/dRep/test/data/res.dRep.txwf.uniq/data/ANImf_files/Bin.012.u.fasta/Bin.012.u.fasta_vs_Bin.018.fasta.ntref 5.07
4: FINISHING DATA
